portal/static/js/initialQueries.js

```diff
@@ -481,6 +452,9 @@
                     case "select":
                         isComplete = field.val() !== "";
                         break;
+                    case "hidden":
```

Comment:
Might be able to save a few lines if "select" and "hidden" [have the same case](https://stackoverflow.com/questions/7559531/javascript-switch-statement-is-there-a-way-for-two-cases-to-run-the-same-code/7559559#7559559)...